readme.txt

```diff
@@ -75,6 +75,7 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
 
 == Unreleased ==
 
+- Dev: Update Github E2E CI parameters
```

Comment:
Oh just one thing we don't need a changelog entry for this, happy if you remove it. If you think its needed though it also has to have a PR number for the lint to pass.

Reply:
@samueljseay Thanks, I just removed the commit with this change from the PR.